.github/workflows/tests.yml

```
@@ -233,6 +235,17 @@ jobs:
         with:
           jobs: ${{ toJSON(needs) }}
 
+      - name: Report failures
+        if: github.event_name == 'schedule' && failure()
+        uses: JasonEtco/create-an-issue@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          RUN_ID: ${{ github.run_id }}
+          TITLE: "[test-bot] Scheduled tests failed"
```

```suggestion
          TITLE: "Scheduled tests failed"
```

Let's not add labels to titles, we could maybe add a new label instead?